signs-work-order-map/types/map.ts

```diff
@@ -0,0 +1,107 @@
+export interface LatLon {
+  latitude: number | undefined;
+  longitude: number | undefined;
+}
+
+export interface MapProps {
+  location: {
+    longitude: number | undefined;
+    latitude: number | undefined;
+  };
+  signs: Sign[];
+}
+
+export interface Sign {
+  id: string;
+  lng: number;
+  lat: number;
+  spatialId: number;
+  workOrderId: string;
+  locationDetailPage: boolean;
+}
+
+export interface KnackRecord {
+  /** spatial id */
+  field_3297: number;
```

Comment:
I like how this serves as documentation of the Knack fields! Is there any possibility of these field numbers changing between prod and the test apps? It has been way too long since I've been in the Knack world. 😅

Reply:
good point, field numbers could stray if fields are being added to prod and then different ones are added to test before test is brought up to match prod. I don't think knack development is happening, but its something to be aware of